packages/message/src/PDRs.ts

```diff
@@ -71,3 +87,83 @@ export const getMessagePdrPANMessage = (
 export const getMessagePdrName = (
   message: MessageWithOptionalPayloadPdr
 ): string | undefined => getMessagePdr(message)?.name;
+
+/**
+ * Get the number of running executions for a PDR, if any.
+ *
+ * @param {MessageWithOptionalPdrStats} message - A workflow message
+ * @returns {number} Number of running executions
+ *
+ * @alias module:PDRs
+ */
+export const getMessagePdrRunningExecutions = (
+  message: MessageWithOptionalPdrStats
+): number => (message.payload?.running ?? []).length;
```

Comment:
```suggestion
): number => (message.payload.running ?? []).length;
```

Reply:
Based on the typing above, these properties are all optional so I think I need the `?`. I'm not sure what guarantee we have about these properties actually existing on the cumulus message received by the Lambda, but it felt safer to be defensive

Reply:
Right, but if they're not defined, they're undefined, assuming we can assume `payload`.     I suppose `null` is the other case here, but even then `null || [] === []` 

It's a nit, so I'll resolve.